### ISSUE TITLE
stellate cli login crash

### ISSUE THREAD

**Issue by rezof** [StellateHQ/cli#4]
**Describe the bug**
`stellate login` command crashes every time, it opens a browser window but at the same time the cli crashes.
```
Authenticating via your browser... (https://stellate.co/cli-auth?code=eb6c193ce08f1655d461ddf5c5d31ca7c6e4d88680f4772222cf858322c2c805)
Error: Cannot read property 'consumeLoginRequest' of undefined
```

**To Reproduce**
1. type `stellate login` into the cli
2. a new browser window is opened with `https://stellate.co/cli-auth?code=ed2a48154da65dc4add9f48996e7480e24e903fd9c7b11ffeb2cf78c2f8b55ad`
3. cli crashes at the moment the window is opened

**Desktop (please complete the following information):**
 - macos
 - i tried v14.17.6 & v18.9.0
 - 1.17.0, i tried also 1.16.0

**Comment by mxstbr** [comment]
Thanks for the report @rezof! We have fixed this, and the update will go live in the next version of the CLI; stay tuned.

**Comment by mxstbr** [comment]
This was resolved in `stellate@1.17.1`! Thanks for the report @rezof
